tools/regenerate-docs.ts

```diff
@@ -0,0 +1,126 @@
+#!/usr/bin/env ts-node-transpile-only
+
+// eslint-disable-next-line strict
```

Comment:
I actually don't know why this is complaining because its fine in `eslint-plugin-jest` and the linting configs are the same - I'm just going to disable it here for now and might look into it another time 🤷